meta/api/request/index.ts

```diff
@@ -12,6 +12,7 @@ export type CycleParams = {
 export type CycleDataParams = CycleParams & {
   // TODO: rename to sectionName
   section: string
+  sectionName: string
```

Comment:
for now, param must be section.
in all calls internally (api / controllers we use sectionName)
renaming this parameter is a separate task,

Reply:
most of the work already done in this and next
urlParam section needs to be renamed to sectionName

Reply:
the middleware uses section. use as url param section for now.
we'll rename it in a next task